### PR TITLE
Static Analyzers Run Button Fix

### LINKED ISSUE #3839: SSA Button tooltip origin fix
Fix the tooltip on the static analysis button so it doesn't originate form the button text also fix message.

### PR DESCRIPTION
**Pull request by joeizang**
fixes #3839

### CODE REVIEW

**Review by Aniket-Engg — COMMENTED** [review]
I think issue is also about to change the tooltip message as per discussion

**Comment by joeizang** [comment]
> I think issue is also about to change the tooltip message as per discussion

This is a different pr @Aniket-Engg. The pr with improvements will be ready soon

File: apps/remix-ide-e2e/src/tests/staticAnalysis.test.ts [hunk 1]
```diff
@@ -33,33 +33,36 @@ module.exports = {
...
+      .assert.containsText('span#ssaRemixtab > *[data-id="RemixStaticAnalysisErrorCount', '382')
+      .click('label[id="headingshowLibWarnings"]')
+      .pause(1000)
+      .assert.containsText('span#ssaRemixtab > *[data-id="RemixStaticAnalysisErrorCount', '1')
+      .end()
   }
 }
 
-function runTests (browser: NightwatchBrowser) {
+function runTests(browser: NightwatchBrowser) {
```
Comment (LianaHus) [inline apps/remix-ide-e2e/src/tests/staticAnalysis.test.ts:65]: you need a space here

File: apps/remix-ide-e2e/src/tests/staticAnalysis.test.ts [hunk 2]
```diff
@@ -72,13 +75,13 @@ function runTests (browser: NightwatchBrowser) {
...
-      '#staticanalysisresult .warning',
-      browser
+        '#staticanalysisresult .warning',
+        browser
       )
     })
 }
 
-function listSelectorContains (textsToFind: string[], selector: string, browser: NightwatchBrowser) {
+function listSelectorContains(textsToFind: string[], selector: string, browser: NightwatchBrowser) {
```
Comment (LianaHus) [inline apps/remix-ide-e2e/src/tests/staticAnalysis.test.ts:84]: you need a space here

File: libs/remix-ui/static-analyser/src/lib/Button/StaticAnalyserButton.tsx [hunk 3]
```diff
@@ -16,31 +16,45 @@ const StaticAnalyserButton = ({
...
+          id="staticAnalysisRunBtn"
+          className={newclassList}
+          disabled={disabled}
+          onClick={onClick}
+          style={{ pointerEvents: 'none', color: 'white' }}
+        >
+          <span className="pl-3 pr-4">
+            {buttonText}
+          </span>
+          </button>
```
Comment (LianaHus) [inline libs/remix-ui/static-analyser/src/lib/Button/StaticAnalyserButton.tsx:54]: indent

**Review by LianaHus — CHANGES_REQUESTED** [review]

**Comment by LianaHus** [comment]
how it was fixed exactly?

**Comment by Aniket-Engg** [comment]
> > I think issue is also about to change the tooltip message as per discussion
> 
> This is a different pr @Aniket-Engg. The pr with improvements will be ready soon

In that case, can you update PR desc explaining the purpose of this PR?  Just to add, merging this  PR will close that issue, so I believe everything in that issue should fixed by this PR

**Review by Aniket-Engg — APPROVED** [review]
LGTM

**Review by LianaHus — APPROVED** [review]

**Pull request merged**